providers/hns/hns_roce_u.h

```diff
@@ -362,6 +368,11 @@ struct hns_roce_u_hw {
 	struct verbs_context_ops hw_ops;
 };
 
+static inline unsigned long ROUND_UP(unsigned long n, unsigned long d)
```

Comment:
I don't see any use of this function. Where do you use it?

Reply:
Thanks, this function is redundant, I will remove it.